app/partials/app.cjsx

```diff
@@ -72,7 +70,7 @@ module.exports = React.createClass
   getEnv: ->
     reg = /\W?env=(\w+)/
     browser_env = window?.location?.search?.match(reg)
-    @state?.env || browser_env || 'staging'
+    @state?.env || browser_env[1] || 'staging'
```

Comment:
No, I'm wrong, the API client doesn't export `env`, just the various API hosts: https://github.com/zooniverse/panoptes-javascript-client/blob/master/lib/config.js#L59-L66

But I should probably fix this so it doesn't bomb out if there is no `env` query parameter in the window location.